modules/vaos/app/serializers/vaos/cc_supported_sites_serializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'fast_jsonapi'
+
+module VAOS
+  class CCSupportedSitesSerializer
+    include FastJsonapi::ObjectSerializer
+
+    set_id :id
```

Comment:
So this is going to be interesting. Even though we get an array, it's not really a collection, right? A collection would imply that each member has an id. In fact, I'm not sure any members have an "id" so maybe, since your parameter that you're passing back essentially returning a constant, which could happens to be an array, we could consider setting the value of ID to that.

Lets record the VCR cassettes and we can revisit this. Maybe there is something in the payload returned that is better. But right now, this doesn't make any sense, since you're not likely to get an "id" in the payload returned to you.

Reply:
i expect to get an id (ie, '983') for each of the objects in the array.  Well, it's called 'id' and it's the site_id so i thought that would be appropriate to use here.

Reply:
I see, in that case you'll probably want to do something like:

set_id :site_id